src/routes/governance/components/proposal-change-table/proposal-change-table.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen } from "@testing-library/react";
+import { format } from "date-fns";
+
+import {
+  ProposalRejectionReason,
+  ProposalState,
+} from "../../../../__generated__/globalTypes";
+import { DATE_FORMAT_DETAILED } from "../../../../lib/date-formats";
+import { generateProposal } from "../../test-helpers/generate-proposals";
+import { ProposalChangeTable } from "./proposal-change-table";
+
+it("Renders with data-testid", () => {
+  const proposal = generateProposal();
+  render(<ProposalChangeTable proposal={proposal} />);
+  expect(screen.getByTestId("proposal-change-table")).toBeInTheDocument();
+});
+
+it("Renders all data for table", () => {
+  const proposal = generateProposal();
+  render(<ProposalChangeTable proposal={proposal} />);
+  expect(screen.queryByText("id")).toBeInTheDocument();
```

Comment:
Dont think it matters too much but in most other test files I think we use screen.getByText to assert that an element is present and queryByText to assert an element is _not_ present (as getByText will error).

That said I always feel like writing `expect(screen.getByText('foo')).toBeInTheDocument()` a little weird as the assertion isn't called if the element isn't found due to it erroring out.